functions/src/events/scrapeEvents.ts

```diff
@@ -138,12 +146,60 @@ class HearingScraper extends EventScraper<HearingListItem, Hearing> {
     return events.filter(HearingListItem.guard)
   }
 
-  async getEvent({ EventId }: HearingListItem) {
-    const content = HearingContent.check(await api.getHearing(EventId))
+  async getEvent({ EventId }: HearingListItem /* e.g. 4962 */) {
+    const data = await api.getHearing(EventId)
+    const content = HearingContent.check(data)
+    const EventInDb = await db.collection("events").doc(String(EventId)).get()
```

Comment:
Just double-checking - does this work as expected? IIRC Firestore `db.collection().doc().get()` returns a `DocumentSnapshot` that you need to call `.data()` on to get to the actual JS object with all the fields

Source: https://firebase.google.com/docs/reference/js/firestore_.documentsnapshot

Reply:
Updated to use document data in the latest commit.